meetings/2020_11_24/README.md

```diff
@@ -0,0 +1,110 @@
+This also becomes a proving ground for NoDL.
+
+Simulation may be challenging; a simulated implementation may not quite match the real world implementation. However, we should be able to spec the project in stages. Start simple and build upon the demo.
+
+### Enable DDS security without a file system
+
+The scope of this issue is much wider than just security. Success depends upon buy-in from both the micro-ROS community and from Open Robotics.
+
+The agrees to continue to move the discussion forward to flesh out a design, but not to perform any work on the code at this time.
+
+### [sros2 quality](https://github.com/ros2/sros2/issues/217)
+
+Even though a quality upgrade is stalled on dependent package quality levels, we should continue working on improving sros2 quality. The most important work is to improve documentation.
+
+Currently sros2 users aren't using online resources, and they need more / better documentation. The recommended path forward is to add a full section on security to the ROS 2 tutorials. This should build on the examples of the existing tutorials, and demonstrate how to re-do them with security enabled.
+
+A discussion also ensued on the current status of [answers.ros.org](https://answers.ros.org/questions/).
+
+### Permissions file size
+
+Mikael has been working on uglifying the permissions files. Work on this continues.
+
+### Integration test failures
+
+Mikael described the current state of [failures in test_security](https://github.com/ros2/system_tests/issues/446). The WG agreed that these tests should be fixed, although no specific action items were identified.
+
+### Conclusion
+The WG will focus on the following primary items for G-turtle:
+
+- A reference implementation of security
+- Improving sros2 quality through documentation updates
+
+The WG will also continue working on the following items:
+
+- Design input for running ROS without a file system
+- Reducing permission file size / complexity
+- Fixing test failures
+
+## Open Discussion
+ROS launch status: the initial launch is working but does not include access control. The work is in progress, but stalled pending discussions on [launch_ros PR 180](https://github.com/ros2/launch_ros/pull/180). Some comments are suggesting a plugin solution, which would change future PRs.
+
+Marco suggested [the Robotics Middleware Framework (RMF)](https://github.com/osrf/rmf_demos) as a reference implementation for ROS security. This should be ready to run with ROS 2; they have already done some work with security as well.
+
+Marco also asked about revoking keys: there's a need to handle that within RMF should an individual robot in a fleet be physically compromised. @JaimeMartin provided [information on CRLs from eProsima](https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/auth_plugin/auth_plugin.html#generating-the-certificate-revocation-list-crl).
```

Comment:
We should use an homogeneous way to refer to individuals in the notes (Github handles, first name, initials...).

Reply:
Agreed--I experimented with github handles, but they don't carry through to markdown. Plan on linking to github profiles in the "attendee" list, but then using first names throughout similar to the style used on the wiki site.